client/src/api/managment-service.ts

```diff
@@ -383,6 +391,28 @@ class ManagmentServiceApi {
     return reports;
   }
 
+  public async getReportsScheduled(): Promise<ReportSummary[]> {
+    await this.refreshTokenIfExpired();
+    const response = await this.axiosInstance.get('/api/v1/reports/scheduled');
+    const reports: ReportSummary[] = response.data;
```

Comment:
Do we need to split it into seperate endpoints (scheduled and ondemand)?

Reply:
more endpoints mean more code 🤑